src/main/java/com/conveyal/r5/analyst/scenario/RemoveStops.java

```diff
@@ -47,6 +48,16 @@
     /** Stops to remove from the routes. */
     public Set<String> stops;
 
+    /**
+     * Number of seconds (in addition to dwell time, if any) to remove from the schedule at each removed stop,
+     * to account for acceleration, deceleration, merging, and so on.
```

Comment:
And of course merging into traffic could be confusing once we have a branch called traffic :) :)